core/src/main/scala/cats/evidence/As.scala

```diff
@@ -34,6 +34,13 @@ sealed abstract class As[-A, +B] extends Serializable {
   @inline final def compose[C](that: (C As A)): (C As B) = As.compose(this, that)
 
   @inline final def coerce(a: A): B = As.witness(this)(a)
+
+  /**
+   * A value `A As B` is always sufficient to produce a similar `Predef.<:<`
+   * value.
+   */
+  @inline final def toPredef: A <:< B =
+    substitute[<:<[*, B]](implicitly[B <:< B])
```

Comment:
I think this needs the same treatment as above? Something like:
```scala
type F[-Z] = Z <:< B
substitute[F](implicitly[B <:< B])
```

Reply:
yes, you are right! I got impatient because on my slow machine the tests took too long. I've fixed the tests on 2.12 and 2.13 so I think it will be green now.

Thank you for the review and thanks for considering it for 2.2.0.